openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -30,20 +30,18 @@ def get_queryset(self):
         Returns:
             Queryset: Active admin users.
         """
-        admin_group_names = [*BusinessLine.objects.values_list('title', flat=True), ADG_ADMIN_GROUP_NAME]
+        admin_group_names = [*BusinessLine.objects.values_list('group__name', flat=True), ADG_ADMIN_GROUP_NAME]
+
+        prefetch_admin_groups = Prefetch(
+            'groups', Group.objects.filter(name__in=admin_group_names), to_attr='admin_groups'
+        )
```

Comment:
While getting the groups using BusinessLine model we can omit `__name` which would then return groups instead of group names, hence, we won't need lines 35 till 36. Also we can use these groups in filter (line 41) by removing `__name` and just filtering on the bases of group instead of group name as well as in prefetch (line 43).